bigquery/unit_tests/test__helpers.py

```diff
@@ -491,6 +491,363 @@ def test_instance_setter_w_bad_udfs(self):
         self.assertEqual(instance.udf_resources, [])
 
 
+class Test_AbstractQueryParameter(unittest.TestCase):
+
+    @staticmethod
+    def _get_target_class():
+        from google.cloud.bigquery._helpers import AbstractQueryParameter
```

Comment:
If we want to adopt the rule that imports-inside-testcases must be followed by a blank line, we should have a monster PR which fixes all the places where we *don't* have one, and then start enforcing it.